client/App.svelte

```diff
@@ -1,9 +1,25 @@
 <script>
-	import TutorialView from "./tutorial/TutorialView.svelte";
-	import HomeView from "./home/HomeView.svelte";
+	import HomeView from "./screens/HomeView.svelte";
+	import JoinView from "./screens/JoinView.svelte";
 	import GameView from "./game/GameView.svelte";
+	import { socket } from "./connectivity.js";
 
 	// can be in setup, tutorial, or playing
+
+	let roomCode = "";
+	let parts = window.location.href.split("?");
+	if (parts.length == 2) {
+		roomCode = parts[1];
+	}
```

Comment:
We might want to use [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) here, it's a little more flexible.
```suggestion
	const roomCode = new URLSearchParams(window.location.search).get("code"); // string | null
```

Reply:
Updated with slight changes